src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,117 @@
+import {
+  faClipboardList,
+  faCloudBolt,
+  faGear,
+  faPencil,
+  faSearch,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useRef, useState } from "react";
+import "./home.css";
+
+export default function Home() {
+  const focusInput = useRef("");
+  const [focusText, setFocusText] = useState("");
+  const [focusState, setFocusState] = useState(false);
+  const [checked, setChecked] = useState(false);
+
+  const [logoutMenu, setLogoutMenu] = useState(false);
+  const [todoDisplay, setTodoDisplay] = useState(false);
```

Comment:
suggestion: try to refactor it and use `useReducer` as you have more than 3 states

Reply:
will do that, thanks!